ocs_ci/utility/reporting.py

```diff
@@ -1,20 +1,28 @@
+import logging
+
 from ocs_ci.framework import config
 
+log = logging.getLogger(__name__)
+
 
-def get_deployment_polarion_id():
+def get_polarion_id(upgrade=False):
     """
     Determine the polarion_id of the deployment or upgrade
 
+    Args:
+        upgrade (bool): get upgrade_id if true, else get deployment_id
+
     Returns:
         str: polarion_id of the deployment or upgrade
 
     """
     polarion_config = config.REPORTING.get('polarion')
     if polarion_config:
-        if config.UPGRADE.get('upgrade'):
-            if config.DEPLOYMENT.get('subscription_plan_approval') == 'Manual':
-                return polarion_config.get('upgrade_manual_id')
-            else:
-                return polarion_config.get('upgrade_auto_id')
+        if upgrade:
+            upgrade_id = polarion_config.get('upgrade_id')
+            log.info('polarion upgrade_id: %s', upgrade_id)
```

Comment:
```suggestion
            log.info(f'polarion upgrade_id: {upgrade_id}')
```

Reply:
Those log messages are in line with our code guidelines https://github.com/red-hat-storage/ocs-ci/blob/a94361d4f30b3a1901ce376f0e85048c26707e78/docs/coding_guidelines.md for Logging.

Reply:
Correct, when it comes to logging we have decided to let the logger handle the message formatting.